packages/web-react/src/components/Breadcrumbs/README.md

```diff
@@ -91,4 +91,9 @@ Use the `BreadcrumbsItem` component for the ordered list as the component's chil
 | `UNSAFE_className` | `string`        | —               | ✕        | Wrapper custom class name                   |
 | `UNSAFE_style`     | `CSSProperties` | —               | ✕        | Wrapper custom style                        |
 
+### Dealing with long titles
+
+When you need to shorten the title of the BreadcrumbsItem you can use both helper class `text-truncate` or use any of the multiple ways in JavaScript which will truncate a string for you like `.slice` or `.substring`, e. g. `str.slice(0, num) + '...'`.
+You can also use any of the existing npm packages which deals with truncating the string.
```

Comment:
```suggestion
You can also use any of the existing npm packages which deal with truncating the string.
```

Maybe we can link the search from npm?
https://www.npmjs.com/search?q=truncate

Reply:
Fixed.